server/sonar-server/src/main/java/org/sonar/server/issue/ws/BulkChangeAction.java

```diff
@@ -206,17 +211,18 @@ public void handle(Request request, Response response) throws Exception {
       issueStorage.save(items);
       items.forEach(sendNotification(issueChangeContext, bulkChangeData));
       buildWebhookIssueChange(bulkChangeData.propertiesByActions)
-        .ifPresent(issueChange -> issueChangeTrigger.onChange(
-          new IssueChangeTrigger.IssueChangeData(
+        .ifPresent(issueChange -> {
+          org.sonar.server.qualitygate.changeevent.QGChangeEventFactory.IssueChangeData issueChangeData = new QGChangeEventFactory.IssueChangeData(
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this fully qualified name with "QGChangeEventFactory" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1942)